test/test_env_setup.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+# This script will create a service account for GCS used for testing and will add to ../.env the necessary env variables.
+# The script below assumes you have run `gcloud auth login`
+#
+# Set the following before running the script:
+#   GOOGLE_CLOUD_PROJECT=[SET YOUR PROJECT ID]
```

Comment:
It seems a good idea to check if this env variable is defined, and exit otherwise.